account/watcher/controller.go

```diff
@@ -26,33 +26,23 @@ type expiryReq struct {
 
 // Config contains all of the Watcher's dependencies in order to carry out its
 // duties.
-type Config struct {
+type CtrlConfig struct {
```

Comment:
nit: Godoc comment doesn't match, same for `NewController`.

Reply:
nit: needs update